src/Umbraco.Core/Migrations/Upgrade/V_8_15_0/UpdateCmsPropertyGroupIdSeed.cs

```diff
@@ -0,0 +1,21 @@
+using NPoco;
+using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.Dtos;
+
+namespace Umbraco.Core.Migrations.Upgrade.V_8_15_0
+{
+    public class UpdateCmsPropertyGroupIdSeed : MigrationBase
+    {
+        public UpdateCmsPropertyGroupIdSeed(IMigrationContext context) : base(context)
+        {
+        }
+
+        public override void Migrate()
+        {
+            if (DatabaseType.IsSqlCe())
```

Comment:
Is it worth performing this SQL statement regardless of DB type ?

Reply:
Since we always use `IDENTITY(1,1)` for SQL Server I honestly don't think it makes sense to make this change to other types than SqlCe, since these are we only two types we support. Should we add support for another database type this won't be necessary since the identity seed on `PropertyTypeGroupDto` has been update 😄